### PR TITLE
Change the apiVersion to apps/v1

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Review by lilic — APPROVED** [review]
Thanks! 👍 

/lgtm

**Comment by rh-max** [comment]
@soltysh Changing apiVersion per your request. The other parts of the docs will do the same shortly.

**Comment by LiliC** [comment]
Thanks for spotting it @soltysh :)

**Comment by rh-max** [comment]
@LiliC Thank you!
@juzhao Could you please review the change?

**Comment by juzhao** [comment]
LGTM if the change is for 4.4 only, 4.3 can still use "extensions/v1beta1"

**Comment by rh-max** [comment]
@juzhao According to @LiliC and @soltysh, while `extensions/v1beta1` _can_ be used in 4.3, it is deprecated and should be avoided.

**Comment by LiliC** [comment]
4.3 should also follow best practices and migrate from using the extensions/v1beta1, so we should avoid mentioning or encouraging users to use it. There are even some ideas of adding alerts to 4.3 that would fire if you are using that api. So switching to both 4.3 and 4.4 is the right thing to do here.

File: modules/monitoring-configuring-monitoring-for-an-application.adoc
Comment (LiliC) [inline modules/monitoring-configuring-monitoring-for-an-application.adoc]: I get the following error when trying to apply this file via the UI in a 4.3 cluster:

> An error occurred
> Error "Required value" for field "spec.selector".

**Review by lilic — CHANGES_REQUESTED** [review]

File: modules/monitoring-configuring-monitoring-for-an-application.adoc
Comment (LiliC) [reply modules/monitoring-configuring-monitoring-for-an-application.adoc]: We need to also update the very last thing to replace it with:

> oc port-forward -n openshift-user-workload-monitoring svc/prometheus-operated 9090

we did not include namespace.

Comment (LiliC) [reply modules/monitoring-configuring-monitoring-for-an-application.adoc]: There was a bug in the app can you also change the image to: `quay.io/brancz/prometheus-example-app:v0.2.0`

This is my final deployment:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: example-app
  namespace: default
spec:
  selector:
    matchLabels:
      app: example-app
  replicas: 1
  template:
    metadata:
      labels:
        app: example-app
    spec:
      containers:
      - name: example-app
        image: quay.io/brancz/prometheus-example-app:v0.2.0
        ports:
        - name: web
          containerPort: 8080
---
```

**Review by soltysh — APPROVED** [review]
lgtm, except the comment from Lili

**Review by lilic — APPROVED** [review]
👍 
/lgtm

**Comment by rh-max** [comment]
Thanks @soltysh for reporting, thanks @LiliC and @juzhao for help.
@bmcelvee, could you

**Comment by bmcelvee** [comment]
LGTM - before merging, I can't seem to find where these are building in the docs. Would you mind linking preview builds?

Thanks! :)

**Review by lilic — COMMENTED** [review]
/lgtm 

👍

**Comment by rh-max** [comment]
@bmcelvee The only updates that go to the published docs are in`modules/monitoring-deploying-a-sample-service.adoc`, as built here https://monitoring-change-apiversion--ocpdocs.netlify.com/openshift-enterprise/latest/monitoring/monitoring-your-own-services.html#deploying-a-sample-service_monitoring-your-own-services (please re-check, contains two total fixes now). The rest of the updates went into the docs that might be published in a future release, but are currently not included and so are not available for a preview.

**Pull request merged**
